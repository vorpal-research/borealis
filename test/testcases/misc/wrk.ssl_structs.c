//
// Created by akhin on 3/14/16.
//

#include <openssl/ssl.h>

typedef struct connection {
    SSL* ssl;
} connection;

int main() {
    connection* c = malloc(sizeof(connection));
    return c->ssl->client_version;
}
