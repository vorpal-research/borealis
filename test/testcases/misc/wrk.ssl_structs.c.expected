[
    {
        "location": {
            "loc": {
                "col": 5,
                "line": 13
            },
            "filename": "test/testcases/misc/wrk.ssl_structs.c"
        },
        "type": "INI-03"
    }
]